ChartIQDemo/ChartIQDemo/StudyDetailViewController.swift

```diff
@@ -241,9 +245,15 @@ class StudyDetailViewController: UITableViewController {
         switch optionType {
         case .select:
             cell.labels?[1].text = (parameter["value"] as? String)?.capitalized ?? ""
-        case .number, .text:
+        case .number, .text, .date, .time:
             cell.textFields![0].keyboardType = optionType == .number ? .numberPad : .default
             cell.textFields?[0].text = optionType == .number ? String(parameter["value"] as? Float ?? 0) : parameter["value"] as? String ?? ""
+            if(optionType == .date) {
+                cell.labels?[0].text! += " (yyyymmdd)"
+            }
+            if(optionType == .time) {
+                cell.labels?[0].text! += " (hh:mm:ss)"
```

Comment:
Should the colons be here?

Reply:
> Looks like the label is used to identify the field. Now it won't save changes to the date & time. If this is going to be a can of worms I think it's best not to pursue seeing as how this app will be deprecated soon.

Heh, of course it breaks it... Good thing you checked. I say revert and just leave the label as is. It's not worth it to make a check on the sdk side.

Reply:
How are we handling these fields in the new app? Is there a datetime picker?